docs/man/dunes.1

```diff
@@ -1,166 +1,166 @@
-.\" Automatically generated by Pandoc 2.5
+.\" Automatically generated by Pandoc 2.9.2.1
 .\"
-.TH "DUNE" "1" "April 04, 2023" "AntelopeIO" "Docker Utilities for Node Execution"
+.TH "DUNES" "1" "April 04, 2023" "AntelopeIO" "Docker Utilities for Node Execution and Subsystems"
 .hy
 .SH NAME
 .PP
-DUNE \- Docker Utilities for Node Execution.
+DUNES - Docker Utilities for Node Execution.
 .SH SYNOPSIS
 .PP
-\f[C]dune <option> [<arguments>] [\-\- <commands>]\f[R]
+\f[C]dunes <option> [<arguments>] [-- <commands>]\f[R]
 .SH DESCRIPTION
 .PP
-\f[B]DUNE\f[R] is a tool to abstract over Leap programs, CDT, and other
+\f[B]DUNES\f[R] is a tool to abstract over Leap programs, CDT, and other
 services and tools to perform the functions of node management,
 compiling smart contracts, running tests, and other tasks required to
 develop smart contracts on Antelope blockchains.
 .SH OPTIONS
 .PP
-\f[B]\f[CB]\-\- <COMMANDS>\f[B]\f[R]
+\f[B]\f[CB]-- <COMMANDS>\f[B]\f[R]
 .IP
 .nf
 \f[C]
 Adding double dashes and space at the end of the command line appends trailing arguments to the current 
 command. If the current command is empty, trailing arguments are passed as though they are the command.
 
-Example: dune \-\- cleos \-\-help
+Example: dunes -- cleos --help
```

Comment:
interesting, @ScottBailey: I thought backslash was needed here, but it seems it works as it is now (both `man` and `md`).

Reply:
This is auto-generated with the tool - and it's different for different versions. Part of why I'm against committing files like this.